beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -191,3 +192,86 @@ func ProcessEjections(state *pb.BeaconState) (*pb.BeaconState, error) {
 	}
 	return state, nil
 }
+
+// ProcessValidatorRegistry computes and sets new validator registry fields,
+// reshuffles shard committees and returns the recomputed state.
+//
+// Spec pseudocode definition:
+//	Set state.validator_registry_latest_change_slot = state.slot.
+//	Set state.shard_committees_at_slots[:EPOCH_LENGTH] = state.shard_committees_at_slots[EPOCH_LENGTH:].
+//	Set state.shard_committees_at_slots[EPOCH_LENGTH:] =
+//  	get_new_shuffling(state.latest_randao_mixes[(state.slot - EPOCH_LENGTH) %
```

Comment:
Can you indent these next 3 lines? It was hard to understand this is an assignment